.github/workflows/actions.yml

```diff
@@ -95,7 +95,7 @@ jobs:
     uses: ./.github/workflows/run-mill-action.yml
     with:
       java-version: '8'
-      millargs: mill.scalalib.scalafmt.ScalafmtModule/checkFormatAll __.sources
```

Comment:
Can we please keep the format check. It's unrelated to unused imports.

Reply:
Fixed